pkg/docs/reference.md

```diff
@@ -38,31 +38,27 @@ rules such as `pkg_files`, and `pkg_mkdirs`.  These allow fine-grained control
 of the contents of your package.  For example:
 
 ```python
-attributes = {
-    "unix": ("0644", "myapp", "myapp"),
-    "my_custom_attribute": ("some custom value",),
-}
+attributes = pkg_attributes(
+    mode = "0644",
+    user = "root",
+    group = "wheel",
+    my_custom_attribute = "some custom value",
+)
 ```
 
-Known attributes include the following:
+`mode`, `user`, and `group` correspond to common UNIX-style filesystem
+permissions.  Attributes should always be specified using the `pkg_attributes`
+helper macro.
 
-#### `unix`
+When `mode` is not specified, it defaults to "0444" (read-only).  If `user` and
+`group` are not specified, then defaults will be chosen by the underlying
+package builder.  Any specific behavior from package builders should not be
+relied upon.
 
-Specifies basic UNIX-style filesystem permissions. For example:
+Any other attributes should be specified as additional arguments to the
+`pkg_attributes` rule.
 
-```python
-{"unix": ("0755", "root", "root")}
-```
-
-This is a three-element tuple, providing the filesystem mode (as an octal
-string, like above), user, and group.
-
-`"-"` may be provided in place of any of the values in the tuple.  Packaging
-rules will set these to a default, which will vary based on the underlying
-package builder and is otherwise unspecified.  Relying upon values set by it is
-not recommended.
-
-Package mapping rules typically set this to `("-", "-", "-")`.
+There are currently no other well-known attributes known at this time.
```

Comment:
"currently .... at this time" is redundant. One or the other.

Reply:
Fixed.  Thanks.